__DEFINES/_macros.dm

```diff
@@ -400,3 +400,6 @@ proc/get_space_area()
 
 // strips all newlines from a string, replacing them with null
 #define STRIP_NEWLINE(S) replacetextEx(S, "\n", null)
+
+#define istransformable(A) istype(A, /atom)
+#define isapperanceeditable(A) istype(A, /atom)
```

Comment:
`isatom(A)` (or `isloc(A)` which is exactly the same thing) is marginally faster
All three of these are not quite correct though since `/area` is a child of `/atom`
I doubt this would ever matter though